pkg/networkmanager/networking.css

```diff
@@ -10,6 +10,10 @@
     --pf-l-gallery--GridTemplateColumns: 1fr;
 }
 
+.networking-page .firewall-switch {
+    padding-left: 1rem;
```

Comment:
Wasn't sure about this one, so I asked @jgiardino for advice, and she suggested this.
Instead of hard-coding this, there is a spacing variable from Patternfly we can use.
`var(--pf-global--spacer--md)`

https://www.patternfly.org/v4/documentation/core/utilities/spacing

Reply:
Done